src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,60 @@
+.stars-rate_container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__star {
+  height: 100%;
+  width: 16px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 13%;
+  background-size: 81%;
+}
+
+.stars {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 2px;
+  padding-right: 2px;
+  width: 100px;
+  height: 16px;
+}
+
+.stars--0 > .stars__star {
+  background-image: url(../images/star.svg);
+  background-position: 8%;
```

Comment:
1. don't repeat yourself (DRY) - you shouldn't set this value to modificator
2. why background-position: 8% and size: 80%? there is a simpler way to display it as necessary (don't forget `background-repeat`)
3. use sass loop for it